library/n2_protocol.hpp

```diff
@@ -8,14 +8,35 @@
 
 namespace ioremap { namespace elliptics { namespace n2 {
 
-struct data_in_file {
-	int fd = -1;
-	off_t local_offset = 0;
-	size_t fsize = 0;
-	int on_exit = 0;
+class data_in_file {
```

Comment:
Sorry, I didn't find a way to construct this object. Have you used it?

Reply:
I didn't use it because I support only cache yet. I tried to support eblob_backend in this branch, but I reverted this overworking. Yes, `data_in_file` could be not-finished.